apps/lend/src/components/ChartOhlcWrapper/types.ts

```diff
@@ -8,3 +10,26 @@ export type LendingMarketTokens = {
     address: string
   }
 } | null
+
+export type PoolActivityProps = {
```

Comment:
I prefer to use Interfaces instead of Types to describe props, even though it's essentially the same thing.
https://www.typescriptlang.org/play/?#example/types-vs-interfaces
```suggestion
export interface PoolActivityProps {
```